xml/System.Text.Json.Serialization/JsonExtensionDataAttribute.xml

```diff
@@ -19,8 +19,24 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>When placed on a property of type <see cref="T:System.Collections.Generic.IDictionary`2" />, provides that any properties that do not have a matching member are added to that dictionary during deserialization and written during serialization.</summary>
+    <remarks>
+      <format type="text/markdown">
+            <![CDATA[
+
+## Remarks
+
+The dictionary's `TKey` value must be <xref:System.String>, and `TValue` must be <xref:System.Text.Json.JsonElement> or <xref:System.Object>.
+
+During deserialization, when using <xref:System.Object>, a "null" JSON value is treated as a `null` object reference, and when using <xref:System.Text.Json.JsonElement>, a "null" is treated as a JsonElement with <xref:System.Text.Json.JsonElement.Type> set to <xref:System.Text.Json.JsonValueType.Null?displayProperty=nameWithType>.
+            
+During serialization, the name of the extension data property is not included in the JSON; the data contained within the extension data is serialized as properties of the JSON object.
+            
+If there is more than one extension property on a type, or it the property is not of the correct type, an <xref:System.InvalidOperationException> is thrown during the first serialization or deserialization of that type.
```

Comment:
I think it would be more clear to say if more than one property has this attribute on it.

```suggestion
If there is more than one property on a type with this extension data attribute, or if the property itself is not of the correct <see cref="T:System.Collections.Generic.IDictionary`2" /> type, an <xref:System.InvalidOperationException> is thrown during the first serialization or deserialization of that type.
```

Reply:
Ok I'll commit this second suggestion.